api/v1alpha1/pattern_types.go

```diff
@@ -70,6 +70,10 @@ type PatternSpec struct {
 
 	// Look for external changes every N minutes
 	// ReconcileMinutes int    `json:"reconcileMinutes,omitempty"`
+
+	// Unseal the Vault automatically. Note: This is *insecure*
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
+	InsecureVaultUnseal bool `json:"insecureVaultUnseal,omitempty"`
```

Comment:
```suggestion
	InsecureManagedVault bool `json:"insecureManagedVault,omitempty"`
```

Reply:
better make sure its optional too

Reply:
I renamed it, but I am not sure how to make it optional?

Reply:
It might need to be a *bool.
As long as it doesn't barf on an old spec that doesn't have it.  Maybe a provide a default somehow?

Reply:
Ok I think I tackled this now and moved to *bool and set a false default in applyDefaults(). I hope I tested everything correctly. I put it in a separate commit at the end to make it easier to review